cmd/authd/daemon/fs.go

```diff
@@ -0,0 +1,25 @@
+package daemon
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"syscall"
+)
+
+// ensureDirWithPerms creates a directory at path if it doesn't exists yet with perm as permissions.
+// If the path exists, it will check it’s a directory with those perms.
```

Comment:
```suggestion
// ensureDirWithPerms creates a directory at path if it doesn't exist yet with perm as permissions.
// If the path exists, it will check if it’s a directory with those perms.
```